public/app/core/logs_model.ts

```diff
@@ -265,7 +267,7 @@ interface LogFields {
  * Converts dataFrames into LogsModel. This involves merging them into one list, sorting them and computing metadata
  * like common labels.
  */
-export function logSeriesToLogsModel(logSeries: DataFrame[]): LogsModel | undefined {
+export function logSeriesToLogsModel(logSeries: DataFrame[], datasourceId?: string): LogsModel | undefined {
```

Comment:
datasourceId is not used anymore, so it can be removed:
![image](https://user-images.githubusercontent.com/30407135/81273957-f080a400-904f-11ea-908f-33c314c42186.png)